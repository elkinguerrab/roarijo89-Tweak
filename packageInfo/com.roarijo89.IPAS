{
	"name": "IPAS",
	"desc_long": "ðŸš¨Sign ipasðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br>",
	"screenshot": null,
	"open": true,

}