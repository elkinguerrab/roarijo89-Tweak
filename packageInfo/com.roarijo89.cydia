{
	"name": "cydia 2020",
	"desc_long": "cydia better UI and more<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br><br>",
	"screenshot": null,
	"open": true,
}