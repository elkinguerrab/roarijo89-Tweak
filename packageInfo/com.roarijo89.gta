{
	"name": "GTA San Andreas",
	"desc_long": "ðŸš¨https://pastebin.com/9saYaFV5ðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br>go on the link<br>",
	"screenshot": null,
	"open": true,

"warning": "Login info <br>Username:Coco <br> Password:cocox",

}