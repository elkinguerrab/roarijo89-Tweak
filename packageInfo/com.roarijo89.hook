{
	"name": "HookedTV",
	"desc_long": "ðŸš¨unlimited subscriptionðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br>unlimited subscription<br>",
	"screenshot": null,
	"open": true,

"warning": "Login info <br>Username:roarijo89 <br> Password:roarijo89",

}