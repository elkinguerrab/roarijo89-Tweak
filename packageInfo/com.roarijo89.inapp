{
	"name": "inapp",
	"desc_long": "ðŸš¨inappurchase made by us is gonna get update alot so u guys can get the inappurchase everðŸš¨<p><p><p>",
	"compatitle": "iOS 13",
	"changelog": "<strong>1.1</strong><br>+ added no reckov<br><strong>1.2</strong><br>+ porn free.<br>",
	"screenshot": null,
	"open": true,
}