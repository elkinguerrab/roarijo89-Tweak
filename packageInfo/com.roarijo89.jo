{
	"name": "Jonny Trigger",
	"desc_long": "ðŸš¨in app purchaseðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br>in app purchase<br>",
	"screenshot": null,
	"open": true,

"warning": "Login info <br>Username:roarijo89 <br> Password:roarijo89",

}