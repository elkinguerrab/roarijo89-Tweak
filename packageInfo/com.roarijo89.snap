{
	"name": "Snapchat",
	"desc_long": "ðŸš¨no jailbreak detection canâ€™t tell the other person u took a screenshot prevents u from getting forced logoutðŸš¨<p><p><p>",
	"compatitle": "iOS 11 12 13",
	"changelog": "<strong>1.0</strong><br>added no jailbreak detection<br>added no screenshots notifications<br>added prevent ban /no force logout<br>",
	"screenshot": null,
	"open": true,

"warning": "Login info <br>Username:Coco <br> Password:cocox",

}